apps/pushes/views/diff.py

```diff
@@ -221,24 +219,74 @@ def diffLines(self, path, action):
                   'entity': entity
                 })
             else:
-                oldval = a_entities[a_map[entity]].val
-                newval = c_entities[c_map[entity]].val
-                if oldval == newval:
+                old_entity = a_entities[a_map[entity]]
+                new_entity = c_entities[c_map[entity]]
+                if old_entity.equals(new_entity):
                     continue
```

Comment:
With this `continue` here, you will skip the rest of the diffing logic if entities are semantically equal but their `vals` are different due to whitespace. Is this desired?